File: Gym_.NET-master/Gym.API/Domain/Models/CardSalesHistory.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace Gym.API.Domain.Models
{

    public class CardSalesHistory
    {
        public int IdSalesHistory { get; set; }
        public int IdUser { get; set; }

        public int IdCard { get; set; }
        public DateTime Date { get; set; }

       [ForeignKey("IdCard")]
       public Card Card { get; set; }

        [ForeignKey("IdUser")]
        public User User { get; set; }
    }
}

File: Gym/app/src/main/java/com/example/gym/models/CardSalesHistory.kt
package com.example.gym.models

import com.google.gson.annotations.SerializedName
import java.math.BigInteger

data class CardSalesHistory (

    @SerializedName("idSalesHistory") val idSalesHistory : Int,
    @SerializedName("idUser") val idUser : Int,
    @SerializedName("idCard") val idCard : Int,
    @SerializedName("date") var date : String
)